pages/common/git-archive-file.md

```diff
@@ -0,0 +1,9 @@
+# git archive-file
+
+> Copy current checked-out branch into a zip archive.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-archive-file>.
+
+- Copy HEAD branch into a zip archive:
```

Comment:
`HEAD` is a pointer to the current branch, check: `cat .git/HEAD`.
Do you want to say `last commit` or copy the current working dir with files edited but not committed yet?

Reply:
git archive-file just take a full copy of the current branch you are checked out on (HEAD).

Reply:
You can say `the HEAD` but no everyone knows that means `the current branch`.
But in this case is not `the current HEAD` because you can be in a tag, or a detached commit, so:
```suggestion
- Archive the current commit into a zip archive:
```

Reply:
I think that should be 

> Archive the current branch

Reply:
Yeah, we have issues regarding this Git studff.  I assume it takes a [commit-ish](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish)?

Reply:
https://github.com/tldr-pages/tldr/pull/5960/commits/c3bbf9be5f81a319e1593a01745fa4b6087c5247

Reply:
Hrm. I would suggest something like this:

```suggestion
- Pack the currently checked out commit into a zip archive:
```

Reply:
A branch is really nothing in it of itself, it's just a pointer to a commit, so @sbrl's suggestion is correct. See [this](https://stackoverflow.blog/2021/04/05/a-look-under-the-hood-how-branches-work-in-git/) for a great explanation.

Reply:
https://github.com/tldr-pages/tldr/pull/5960/commits/5284ad6fa74e5e1e6e9c35776c2de5818f794f1f